src/MapKit/MKEnums.cs

```diff
@@ -112,7 +112,14 @@ public enum MKPinAnnotationColor : ulong {
 	[NoWatch]
 	[Native]
 	public enum MKUserTrackingMode : ulong {
-		None, Follow, FollowWithHeading
+		None,
+		Follow, 
+#if !XAMCORE_5_0 && !(IOS || MACCATALYST)
+		[Obsolete ("This is only available on iOS and MacCatalyst")]
+		FollowWithHeading,
+#elif IOS || MACCATALYST
+		FollowWithHeading,
+#endif
```

Comment:
You could have made your life a little simpler:
```suggestion
#if !XAMCORE_5_0 && !(IOS || MACCATALYST)
		[Obsolete ("This is only available on iOS and MacCatalyst")]
#endif
		FollowWithHeading,
```

Reply:
Do we want to keep the enum value available? I think the current fix is ok

Reply:
@mandel-macaque the enum value is only available on iOS and MacCatalyst, removing the `#elif` would make the value available for all platforms on Xamcore 5